src/index.html

```diff
@@ -1,12 +1,29 @@
 <!doctype html>
 <html lang="en">
 <head>
-  <meta charset="utf-8">
+  <meta charset="UTF-8" />
   <title>Systers Open Source</title>
   <base href="/">
-
-  <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <meta charset="UTF-8" />
+  <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+  <meta name="keywords" content="slack, slack-ghc, AnitaB.org, systers, code,
```

Comment:
@ExactlyMay - are there any marketing guidelines for the keywords and description for the systers.io website?

Reply:
Yes if there any such guidelines please do let me know.